README.md

```diff
@@ -12,6 +12,13 @@
 
 Node: `>=8, <9`
 
+### nvm 
+[install nvm ](https://github.com/creationix/nvm)
+
+**Note**:if zsh has been installed in your Mac,you need to install nvm as follow. 
```

Comment:
:bulb: **Note**: If zsh has been set as default shell on your Mac( `echo $0` will see which shell you are using ), you need to install nvm as follow: